cluster-autoscaler/cloudprovider/clusterapi/clusterapi_nodegroup_test.go

```diff
@@ -648,6 +663,23 @@ func TestNodeGroupDecreaseTargetSize(t *testing.T) {
 			includeFailedMachine:               true,
 			includeFailedMachineWithProviderID: true,
 		},
+		{
+			description:                         "A node group with 4 replicas with one pending machine that has a provider ID should decrease by 1",
+			initial:                             4,
+			targetSizeIncrement:                 0,
+			expected:                            3,
+			delta:                               -1,
+			includePendingMachine:               true,
+			includePendingMachineWithProviderID: true,
+		},
+		{
+			description:                  "A node group with target size 4 but only 3 existing instances without provider IDs should decrease by 1",
```

Comment:
let's s/should decrease by 1/should not scale out/

Reply:
updated